src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -100,6 +103,14 @@ const AccessibilityRequestDetailPage = () => {
                   <h2 className="margin-top-2 margin-bottom-3">
                     Test Dates and Scores
                   </h2>
+                  {testDates.map((testDate, index) => (
+                    <TestDateCard
+                      date={DateTime.fromISO(testDate.date)}
+                      type={testDate.testType}
+                      testIndex={index + 1}
+                      score={testDate.score || undefined}
```

Comment:
I would prefer updating the `TestDateCard` props to allow `null` instead of passing in `undefined` manualy.

```
type TestDateCardProps = {
  date: DateTime;
  type: 'INITIAL' | 'REMEDIATION';
  testIndex: number;
  score?: number | null; // A whole number representing tenths of a percent
};
```